src/logging.c

```diff
@@ -1297,7 +1306,6 @@ void rcutils_logging_console_output_handler(
     .allocator = g_rcutils_logging_allocator
   };
 
-  char output_buf[1024] = "";
```

Comment:
These changes shall be done in the original repo: https://github.com/ros2/rcutils